### ISSUE TITLE
Site reloads each time a new page is clicked on. Would love to fix that with react-router

### ISSUE THREAD

**Issue by Psalmuel01** [vikhyatsingh123/Naruto-Shippuden#1133]
I'd love to use react-router to make it a single page application and enable seamless transition between the home page and about page for example, without needing to reloading over and over.

**Comment by vikhyatsingh123** [comment]
@Psalmuel01 sure

**Comment by Psalmuel01** [comment]
On it.

**Comment by Psalmuel01** [comment]
just realized the project is not a react app. do you mind converting it all into one

**Comment by vikhyatsingh123** [comment]
if you can convert it into react, please I will be happy to merge it

**Comment by Psalmuel01** [comment]
yes, i can, but it will be a breaking change. and the whole mode of adding an hero/villain will have to change. i'mma wait till ther'es no more contributions in view to effect it. give me a go ahead when/if ready

**Comment by vikhyatsingh123** [comment]
now, you can start working on it. there will be no more change

**Comment by Psalmuel01** [comment]
okay

**Comment by vikhyatsingh123** [comment]
@Psalmuel01 are you working on it or not?

**Comment by Psalmuel01** [comment]
hey, so sorry for the delay
already started working on it, but got caught up on work
will continue as soon as i get free

**Comment by vikhyatsingh123** [comment]
ok
